Firestore/Source/API/FSTUserDataReader.mm

```diff
@@ -303,8 +303,10 @@ - (ParsedUpdateData)parsedUpdateData:(id)input {
 
 - (Message<google_firestore_v1_Value>)parseDictionary:(NSDictionary<NSString *, id> *)dict
                                               context:(ParseContext &&)context {
-  __block Message<google_firestore_v1_Value> result{};
+  __block Message<google_firestore_v1_Value> result;
   result->which_value_type = google_firestore_v1_Value_map_value_tag;
+  result->map_value.fields_count = 0;
```

Comment:
Optional: you could use list initialization here (`result->map_value = {};`). Totally up to your preference.

Reply:
Fixed